renovate.json

```diff
@@ -18,11 +21,23 @@
         "**/__fixtures__/**"
     ],
     "ignoreDeps": ["neo4j"],
+    "separateMajorMinor": true,
+    "groupName": "all",
```

Comment:
What's the deal with `groupName` out here as well as inside `packageRules`?

Reply:
yeah that's probably not necessary! I looked at this with Liam and we both thought that it was needed but I can't find it written in their docs anywhere today. I'll remove the top level one.